### PR TITLE
Prefer std:find over a hand-coded loop

### PR DESCRIPTION
**Pull request by vondele**
No functional change.

### CODE REVIEW

**Comment by snicolet** [comment]
Any data to back-up this pull request?

**Comment by vondele** [comment]
@snicolet just a non-functional cleanup. Do you suggest to run a [-3,1] ?

**Comment by snicolet** [comment]
Yes, please :-)

**Comment by vondele** [comment]
in queue

File: src/movegen.h [hunk 1]
```diff
@@ -21,6 +21,8 @@
 #ifndef MOVEGEN_H_INCLUDED
 #define MOVEGEN_H_INCLUDED
 
+#include <algorithm>
+
```
Comment (snicolet) [inline src/movegen.h]: This line could be removed

Comment (vondele) [inline src/movegen.h]: @snicolet thanks for the review. 

I'm following the style used throughout the code where there is a blank line between C++ system headers and project headers. That's a common style as well (e.g. https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes )

**Pull request closed without merging**